lib/node_modules/@stdlib/stats/base/dists/triangular/ctor/benchmark/benchmark.js

```diff
@@ -152,17 +167,21 @@ bench( pkg+'::set:b', function benchmark( bm ) {
 	b = 40.0;
 	c = 30.0;
 	dist = new Triangular( a, b, c );
+	len = 100;
+	y = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		y[ i ] = uniform( c + EPS, c + 100.0);
+	}
 
 	bm.tic();
 	for ( i = 0; i < bm.iterations; i++ ) {
-		y = ( 100.0*randu() ) + c + EPS;
-		dist.b = y;
-		if ( dist.b !== y ) {
+		dist.b = y[ i % len ];
+		if ( dist.b !== y[ i % len ] ) {
 			bm.fail( 'should return set value' );
 		}
 	}
 	bm.toc();
-	if ( isnan( y ) ) {
+	if ( isnan( y[ i % len ] ) ) {
```

Comment:
```suggestion
	if ( isnan( y ) ) {
```

This checks whether the random number generation was successful.